config.go

```diff
@@ -249,6 +250,8 @@ func xmlToFileLogWriter(filename string, props []xmlProperty, enabled bool) (*Fi
 			daily = strings.Trim(prop.Value, " \r\n") != "false"
 		case "rotate":
 			rotate = strings.Trim(prop.Value, " \r\n") != "false"
+		case "integersuffix":
+			integerSuffix = strings.Trim(prop.Value, " \r\n") != "false"
```

Comment:
Unfortunately we have to retain the default behavior, and not just make it available. A good test is, if user upgraded the agent and didn't change their configuration file would they still see the same behavior.

Is there a way we can clean up the integer suffix so it's less of a threat to agent health?  I know it sucks to have to clean up the implementation we actively want to get rid of, but it's unfortunately necessary for the backwards compatibility promises we make to our customers.